### PR TITLE
Date- & Timepicker in Config

### PR DESCRIPTION
**Pull request by jsangmeister**

### CODE REVIEW

**Comment by jsangmeister** [comment]
I exchanged OpenSlidesDateAdapter (which is based on NativeDateAdapter) for MomentDateAdapter for localisation, since NativeDateAdapter only supports "en-GB". Please check if this does cause any problems (I haven't found any). (@FinnStutzenstein @tsiegleauq)

File: client/src/app/shared/shared.module.ts [hunk 1]
```diff
@@ -279,7 +292,12 @@ import { ProgressSnackBarComponent } from './components/progress-snack-bar/progr
         ProgressSnackBarComponent
     ],
     providers: [
-        { provide: DateAdapter, useClass: OpenSlidesDateAdapter },
+        // { provide: DateAdapter, useClass: OpenSlidesDateAdapter },  // see remarks in OpenSlidesDateAdapter
```
Comment (tsiegleauq) [inline client/src/app/shared/shared.module.ts]: remove

File: client/src/app/site/config/components/config-field/config-field.component.ts [hunk 2]
```diff
@@ -116,9 +124,11 @@ export class ConfigFieldComponent extends BaseComponent implements OnInit {
      */
     public ngOnInit(): void {
         this.form = this.formBuilder.group({
-            value: ['']
+            value: [''],
+            value2: [''] // used by datepicker to store the time
         });
         this.translatedValue = this.configItem.value;
+        let val2 = null;
```
Comment (tsiegleauq) [inline client/src/app/site/config/components/config-field/config-field.component.ts]: bad name for a variable

**Comment by tsiegleauq** [comment]
Adressed in #4950

**Pull request closed without merging**